app/src/features/projects/view/ProjectPage.tsx

```diff
@@ -111,6 +118,29 @@ const ProjectPage: React.FC = () => {
     onYes: () => dialogContext.setYesNoDialog({ open: false })
   };
 
+  const publishProject = async (publish: boolean) => {
```

Comment:
It looks like the publish param is never actually used, as the boolean is being determine based on the publish_date (line 129). I think we should use the parameter though.  That way the api call being made is more deterministic, and anyone can add a publish or unpublish button to the page, without worrying about the real state of the records publish_date.